greeter_plugin/greeter_summary.py

```diff
@@ -59,7 +59,7 @@ def op(name,
       summary_description=description,
       plugin_data=tf.SummaryMetadata.PluginData(
           plugin_name=PLUGIN_NAME,
-          content=''))
+          content=b''))
```

Comment:
Passing `content=b''` is equivalent to not passing it at all since it's the default value of the protobuf field - the resulting object is the exact same object:

```py
>>> tf.SummaryMetadata.PluginData(plugin_name='foo') == tf.SummaryMetadata.PluginData(plugin_name='foo', content=b'')
True
```

So maybe let's just omit `content` entirely here?  Same goes below.

Reply:
Thank you for the suggestion! I checked back with the protobuf compiler code and you are right, [fields of type `bytes` default to `b''`](https://github.com/google/protobuf/blob/8e44a86facd0f42af7a3c0c47f8133f78f037269/python/google/protobuf/descriptor_pool.py#L903). Removing the `content` parameter from both constructor calls works just as well. Another upside to this might be backwards compatibility for Tensorflow versions older than 1.4.0.

However, this repository is meant to explain how plugins work and the comments above both constructor calls explain that we would be able to pass additional metadata but don't do so in this simple example. Leaving an empty `content` parameter might be a starting point for plugin developers.

So I'm not sure if we should drop the parameter altogether to make the code more concise. A compromise could be dropping the parameter but extending the comments above both calls, explaining that one could use the `content` parameter to pass additional metadata. What do you think?

Reply:
Thanks for asking, that is a fair point.  I think expanding the comment would be an easy way to address this for now.

At some point we should probably elaborate on this example a little bit to actually show some reason why one might want to use the `content` parameter independently of the actual tensor value itself (e.g. it's metadata on the overall tag series, versus a specific item in that series).  But until we're actually showing how you could use the `content` parameter, I think it's reasonable to omit it so that people who don't need it don't get the wrong impression that it's required.

Reply:
Sounds good, I will change the file accordingly.